### ISSUE TITLE
Password entry for some websites stopped working in newer versions of Bitwarden client for Chrome / Edge

### ISSUE THREAD

**Issue by chungr** [bitwarden/clients#7271]
### Steps To Reproduce

1. Go to [https://disneyworld.disney.go.com/plan](https://disneyworld.disney.go.com/plan)
2. Type in your email address
3. Click Continue
4. When prompted for your password, click on the Bitwarden extension, and click on a matching login - the password should be entered for you, however in the latest version of bitwarden it does not.  Going back to version 2023.9.2, the password will be filled in after clicking on the login. 

### Expected Result

1. Go to [https://disneyworld.disney.go.com/plan](https://disneyworld.disney.go.com/plan)
2. Type in your email address
3. Click Continue
4. When prompted for your password, click on the Bitwarden extension, and click on a matching login
5. The password is entered for you
6. In version 2023.9.2, this behavior works as expected - the password will be filled in after clicking on the login. 

### Actual Result

When following the steps to reproduce, the password is never entered for you and you must copy/paste the password.

### Additional Context

This worked properly in 2023.9.2 so I do not believe this is simply a request to update the autofill capability rather this is a bug that was introduced in a recent version of bitwarden.  This also is not simply an autofill ask, as I am clicking on the login first which is separate from the autofill on page load option.

NOTE: This issue also happens on the latest version of Chrome (120.0.6099.109 (Official Build) (arm64)).

### Operating System

macOS

### Operating System Version

13.6.3

### Web Browser

Microsoft Edge

### Browser Version

120.0.2210.77 (Official build) (arm64)

### Build Version

2023.12.1

### Issue Tracking Info

- [X] I understand that work is tracked outside of Github. A PR will be linked to this issue should one be opened to address it, but Bitwarden doesn't use fields like "assigned", "milestone", or "project" to track progress.

**Comment by Krychaz** [comment]
Hello there,

Trying to replicate this. Having the URI set to https://www.disneyworld.com the items autofill as expected on version 2023.12.

Can you please share your URI format and which URL detection you have set?

**Comment by chungr** [comment]
URI is set to disneyworld.disney.go.com which is what you get re-directed to when using the above URL.

URI detection is set to Base Domain and Autofill is set to Auto-fill on Page Load.

The initial username gets autofilled with no problem, but after clicking "Continue", when you are prompted for a password, I cannot get Bitwarden to fill in the password.  In the most recent version, I must copy/paste the password.  However in version  2023.9.2, auto-fill on page load does not always work, but I can always click the bitwarden extension and click on the matching login for the password to be filled in by bitwarden.

**Comment by chungr** [comment]
The latest Bitwarden 2023.12.1 still exhibits the same issue.

I also created a brand new vault just to make sure I did not have any settings wrong, the only thing I changed was autofill on page load and override browser auto-fill settings are both enabled.

URI is set to https://disneyworld.disney.go.com

Detection is set to default, (base domain).

When going to this URL: https://disneyworld.disney.go.com/login?returnUrl=%2Fplan%2F

The autofill works for username, but when I click on Continue, it does not autofill the password.  The expected behavior is that I should be able to at least click on the extension icon, click on the matching login and it will autofill the password.

If it helps, in the JavaScript console I get 4 of the exact same error messages when I click on the matching login and I expect the password would be autofilled, this is a copy/paste from one of those 4 JavaScript console errors:

`More than one element found with opid __0
getAutofillFieldElementByOpid @ bootstrap-autofill.js:176
handleClickOnFieldByOpidAction @ bootstrap-autofill.js:1295
click_on_opid @ bootstrap-autofill.js:1170
(anonymous) @ bootstrap-autofill.js:1189
setTimeout (async)
(anonymous) @ bootstrap-autofill.js:1188
InsertAutofillContentService.runFillScriptAction @ bootstrap-autofill.js:1188
(anonymous) @ bootstrap-autofill.js:1212
(anonymous) @ bootstrap-autofill.js:1158
insert_autofill_content_service_awaiter @ bootstrap-autofill.js:1154
fillForm @ bootstrap-autofill.js:1205
(anonymous) @ bootstrap-autofill.js:1565
(anonymous) @ bootstrap-autofill.js:1469
autofill_init_awaiter @ bootstrap-autofill.js:1465
fillForm @ bootstrap-autofill.js:1560
fillForm @ bootstrap-autofill.js:1486
AutofillInit.handleExtensionMessage @ bootstrap-autofill.js:1508
Show 16 more frames
Show less
`
